build-configs/configs/BuildConfigType.js

```diff
@@ -8,6 +8,34 @@ export type FeatureFlagsType = {|
   introSlides: boolean
 |}
 
+type AndroidGoogleServicesConfig = {|
+  google_app_id: string,
+  gcm_defaultSenderId: string,
+  default_web_client_id: string,
+  ga_trackingId: null | string,
+  firebase_database_url: string,
+  google_api_key: string,
+  google_crash_reporting_api_key: string,
+  project_id: string
+|}
+
+type AppleGoogleServicesConfig = {
+  CLIENT_ID: string,
+  REVERSED_CLIENT_ID: string,
+  API_KEY: string,
+  GCM_SENDER_ID: string,
+  PLIST_VERSION: string,
+  BUNDLE_ID: string,
+  PROJECT_ID: string,
+  STORAGE_BUCKET: string,
```

Comment:
Why do we need this?

Reply:
I don't know. This is the information from the plist. Removing stuff here is probably not supported.